eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -54,10 +68,61 @@ describe("AttendanceCard", () => {
     });
     test("should disable attendance button while state and fb are synching", () => {
       screen.getByText("👎").click();
-      expect(screen.getByTestId("SaulGoodman")).toHaveProperty(
+      expect(screen.getByTestId("Saul-Goodman")).toHaveProperty(
```

Comment:
please. please use snake-case for testids

Reply:
I use the first and last name props which have a capital first letter by default

Reply:
You don't need unique testids as you only need one customer for the testing. If you want to make sure that multiple customers render all the same (which is unnecessary in most cases), you can render multiple customers for that case.

Reply:
The pure react-testing-library way would avoid testids altogether (and only test the stuff the user can see), but you can't really achieve TDD and full coverage that way. That's why I like to use testids, but that should be kept as simple as possible...and in most (99%) cases  you they should be explicitly set.